src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -4333,6 +4333,13 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 
     </GenerateApplicationManifest>
 
+    <AddToWin32Manifest
```

Comment:
hi @sujitnayak, could you please check if the change is correct for ClickOnce manifest?

Reply:
@YuliiaKovalova I am not sure why this needs to be written into ClickOnce's application manifest. ClickOnce's application manifest is different from win32 manifest and has a different schema. ClickOnce runtime that processes this manifest might not be expecting any changes you're making to the manifest.
Can you point me to a before and after of the application manifest with these changes. I would need to check with the ClickOnce runtime team for the impact of these changes.
Thanks.

Reply:
@sujitnayak these targets use a bit of unfortunate nomenclature with `ApplicationManifest` representing the Win32 manifest (`<appname>.exe.manifest`) rather than the ClickOnce application manifest. This PR does not change the ClickOnce manifest.